### PR TITLE
Add poshlib 1.3.002

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **poshlib/1.3.002**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/poshlib/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        license_content = []
+        for i in range(27, 59):
+            license_content.append(content_lines[i][-1])
+        tools.save("LICENSE", "\n".join(license_content))
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self._extract_license()
+        self.copy(pattern="LICENSE", dst="licenses")
```
Comment (uilianries) [inline recipes/poshlib/all/conanfile.py:73]:
```suggestion

    def package(self):
        cmake = self._configure_cmake()
        cmake.install()
        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
```

https://github.com/PhilipLudington/poshlib/blob/master/LICENSE

File: recipes/poshlib/all/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.12)
```
Comment (jgsogo) [inline recipes/poshlib/all/CMakeLists.txt:1]: Probably requires `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` requires higher CMake version, maybe `3.4`?

File: recipes/poshlib/all/conandata.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.3.002":
```
Comment (jgsogo) [inline recipes/poshlib/all/conandata.yml:2]: This is not exactly the version from the origin source, it is `1.3.002` plus some additions, we should use `cci.<date>` instead.

Comment (intelligide) [reply recipes/poshlib/all/conandata.yml:2]: The source codes have not been changed. Only the readme and the license have been modified.

**Review by jgsogo — APPROVED** [review]

File: recipes/poshlib/all/conandata.yml (hunk 3 above)
Comment (uilianries) [reply recipes/poshlib/all/conandata.yml:2]: @jgsogo The author uses "internal version" if you read the readme, but I agree with you, it's fragile in terms of tracking. I just opened an issue asking for a formal tag, so we can download the specific version: https://github.com/PhilipLudington/poshlib/issues/1

**Review by SSE4 — DISMISSED** [review]

File: recipes/poshlib/all/conandata.yml (hunk 3 above)
Comment (uilianries) [reply recipes/poshlib/all/conandata.yml:2]: Now we have a real tag: https://github.com/PhilipLudington/poshlib/releases/tag/v1.3.002

**Review by uilianries — CHANGES_REQUESTED** [review]

File: recipes/poshlib/all/conandata.yml (hunk 3 above)
Comment (intelligide) [reply recipes/poshlib/all/conandata.yml:2]: Thanks !

**Review by danimtb — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]
Thanks!

**Review by uilianries — APPROVED** [review]

**Pull request merged**